### PR TITLE
TextEditor: Add placeholder id to aria-labelledby

### PR DESCRIPTION
**Pull request by marker-dao**

### CODE REVIEW

File: testing/tests/DevExpress.ui.widgets.editors/textEditorParts/common.tests.js [hunk 1]
```diff
@@ -662,32 +662,59 @@ QUnit.module('check aria-labelledby attribute', {
...
+                placeholder: 'custom',
             })
             .dxTextEditor('instance');
         this.$input = this.$textEditor.find(`.${INPUT_CLASS}`);
         this.$label = this.$textEditor.find(`.${LABEL_CLASS}`);
+        this.$placeholder = this.$textEditor.find(`.${PLACEHOLDER_CLASS}`);
     }
 }, () => {
-    QUnit.test('if label is defined', function(assert) {
+    QUnit.test('if label and placeholder are defined', function(assert) {
```
Comment (Krijovnick) [inline testing/tests/DevExpress.ui.widgets.editors/textEditorParts/common.tests.js]: What we expect from the test? It is not clear from the name test.
Fix the tests names, pls

File: testing/tests/DevExpress.ui.widgets.form/formLayoutManager.markup.tests.js [hunk 2]
```diff
@@ -2450,7 +2452,9 @@ QUnit.module('Accessibility', () => {
                     assert.strictEqual($ariaTarget.attr('aria-labelledby'), $label.attr('id'), 'attributes aria-labelledby and labelID are equal');
                 }
             } else {
-                assert.equal($ariaTarget.eq(0).attr('aria-labelledby'), null, `aria-labeledby attribute ${editorClassName}`);
+                const expectedTargetValue = $placeholder.length ? $placeholder.attr('id') : null;
+
+                assert.equal($ariaTarget.eq(0).attr('aria-labelledby'), expectedTargetValue, `aria-labeledby attribute ${editorClassName}`);
```
Comment (ksercs) [inline testing/tests/DevExpress.ui.widgets.form/formLayoutManager.markup.tests.js]:
```suggestion
                assert.equal($ariaTarget.eq(0).attr('aria-labelledby'), expectedTargetValue, `aria-labelledby attribute ${editorClassName}`);
```

File: testing/tests/DevExpress.ui.widgets.editors/textEditorParts/common.tests.js (hunk 1 above)
Comment (marker-dao) [reply testing/tests/DevExpress.ui.widgets.editors/textEditorParts/common.tests.js]: [c05d065c557670099eaf3d0b77467a0ac6ccd2e1].

File: testing/tests/DevExpress.ui.widgets.form/formLayoutManager.markup.tests.js (hunk 2 above)
Comment (marker-dao) [reply testing/tests/DevExpress.ui.widgets.form/formLayoutManager.markup.tests.js]: [c05d065c557670099eaf3d0b77467a0ac6ccd2e1].

**Review by ksercs — DISMISSED** [review]

**Review by Krijovnick — APPROVED** [review]

**Review by ksercs — APPROVED** [review]

**Pull request merged**